api/src/identity-access-management/domain/models/ClaimsMapper.js

```diff
@@ -0,0 +1,50 @@
+import flatten from 'lodash/flatten.js';
+import pick from 'lodash/pick.js';
+
+import { DEFAULT_CLAIMS_MAPPING } from '../constants/oidc-identity-providers.js';
+
+const DEFAULT_OPTIONS = { mapping: DEFAULT_CLAIMS_MAPPING, additionalClaims: [] };
+
+export class ClaimsMapper {
+  constructor(options) {
```

Comment:
La présence des différentes propriétés dans le constructeur sera plus explicite : 

```suggestion
  constructor({ mapping = {}, additionalClaims = [] }) {
```